### DISCUSSION TITLE
Small error in addIterativeLSI

### DISCUSSION THREAD

**Discussion by Austin-s-h** [GreenleafLab/ArchR#669] [General]
Not enough to write a bug report over (and I don't have the faintest idea why this is happening), but even with force = T, addIterativeLSI will fail during the saving of first embedding. 
Deleting the previously saved .RDS and .PDF files in the save folder of the project fixes the issue.

**Comment by rcorces** [comment]
I'm actually not able to recapitulate this error with the tutorial dataset.

**Reply by rcorces** [reply]
That being said, feel free to report any error, regardless of severity under "issues" so that we dont lose track of it.
